.github/workflows/cd-server.yml

```diff
@@ -0,0 +1,54 @@
+name: Server CD
+
+on:
+  push:
+    branches: [dev]
```

Comment:
Maybe we should add `main` branch as well

Reply:
Currently, we use server deploy to integrate client with api, so I think deploy on push in main is not mandatory. This can lead to rewriting between the production and development servers. In the future it will be better to make 2 different deployments for dev and main